### PR TITLE
Fix for 2 unresolved references

### PR DESCRIPTION
**Pull request by ysuarez**
## Link to Github issue or other discussion

#659 

## What does this PR do?

It fixes two unresolved reference errors that Pycharm found in the code.

## What changes were made?

I updated two uses of variables so that they would reference a valid variable. 

## How to test / verify this PR?

I am not currently 100% sure how to properly trigger the code to show that the relevant 2 lines code are now fixed.

* You need to follow a workflow that calls the function `create_children_from_directory()` that then has a `config.yml` file that calls workbench to "Execute media-specific post-create scripts" (and assumes the script(s) are defined), which then calls the line of code in question.
* You need to follow a workflow that calls the function `download_file_from_drupal()`, but in the API call inside you need the server to return an HTTP status that is not 200. 

## Interested Parties

@mjordan

---

## Checklist

* [ x] Before opening this PR, have you opened an issue explaining what you want to to do?
* [ x] Have you run `pycodestyle --show-source --show-pep8 --ignore=E402,W504 --max-line-length=200 yourfile.py`? 
* [ ] Have you included some configuration and/or CSV files useful for testing this PR?
* [ ] Have you written unit or integration tests if applicable?
* [ ] Does the code added in this PR require a version of Python that is higher than the current minimum version?
* [ ] If the changes in this PR require an additional Python library, have you included it in `setup.py`?
* [ ] If the changes in this PR add a new configuration option, have you provided a default for when the option is not present in the .yml file?

### CODE REVIEW

File: workbench_utils.py [hunk 1]
```diff
@@ -5614,7 +5614,7 @@ def create_children_from_directory(config, parent_csv_record, parent_node_id):
         # Execute node-specific post-create scripts, if any are configured.
         if 'node_post_create' in config and len(config['node_post_create']) > 0:
             for command in config['node_post_create']:
-                post_task_output, post_task_return_code = execute_entity_post_task_script(command, args.config, node_response.status_code, node_response.text)
+                post_task_output, post_task_return_code = execute_entity_post_task_script(command, config, node_response.status_code, node_response.text)
                 if post_task_return_code == 0:
```
Comment (mjordan) [inline workbench_utils.py:5618]: `args.config` is intentional here - it's the name of the current Workbench config file.

File: workbench_utils.py [hunk 2]
```diff
@@ -6189,7 +6189,7 @@ def download_file_from_drupal(config, node_id):
                 else:
                     continue
     else:
-        logging.error(f'Attempt to fetch media list {media_list_url} returned an {r.status_code} HTTP response.')
+        logging.error(f'Attempt to fetch media list {media_list_url} returned an {media_list_response.status_code} HTTP response.')
```
Comment (mjordan) [inline workbench_utils.py:6192]: Yes, thanks for catching that. I'll need to test with the updates variable name before merging though.

Comment (mjordan) [reply workbench_utils.py:6192]: Thank for catching that one @ysuarez. I can fix it off-PR if you want and credit you, or you can update this PR by removing the other proposed fix, if you prefer. Let me know!

File: workbench_utils.py (hunk 1 above)
Comment (ysuarez) [reply workbench_utils.py:5618]: I checked again, and of course I may be wrong, but this `args` variable in question is not defined or in scope in line 5617 of `workbench_utils.py` file... 

`post_task_output, post_task_return_code = execute_entity_post_task_script(command, args.config, node_response.status_code, node_response.text)`

or in its enclosing function `create_children_from_directory()`, instead this function signature  accepts a `config` param
https://github.com/mjordan/islandora_workbench/blob/c3fe7e13e1ac16cd5a731555edaf376639b5513a/workbench_utils.py#L5468

here is where `create_children_from_directory()` is called from the `workbench` file with the `config` param set, 
https://github.com/mjordan/islandora_workbench/blob/c3fe7e13e1ac16cd5a731555edaf376639b5513a/workbench#L403-L405

of course in the `workbench` file the `args` variable is defined and in global scope, but it is not in scope within the `workbench_utils.py` file and its code.

Again, I probably would never have noticed this without using the PyCharm linter, but I may still be wrong. 

If you still prefer only the other line change, I will update the PR to only have the other change for you to merge.

Comment (ysuarez) [reply workbench_utils.py:5618]: Yes, I would like to update the PR with your suggested fix. I will put in a new commit in my PR today.

**Comment by ysuarez** [comment]
I just made the update to the PR. 

Mark does Github alert you automatically that I made that update? Just curious about Github notifications.

**Comment by mjordan** [comment]
Yes, I get notifications.

**Comment by mjordan** [comment]
Thanks @ysuarez !

**Pull request merged**